src/main/groovy/com/sequenceiq/ambari/client/services/ClusterService.groovy

```diff
@@ -218,6 +218,31 @@ trait ClusterService extends CommonService {
     })
   }
 
+  /**
+   * Retire a host component on a given host.
+   * @param host hostName where the component is installed to
+   * @param serviceName the service to retire
+   * @param componentName the name of the component
+   */
+  def int retire(List<String> hosts, String serviceName, String componentName)
+          throws URISyntaxException, ClientProtocolException, HttpResponseException, IOException {
+    def requestInfo = [
+            command   : 'RETIRE',
+            context   : "Execute retire command.",
+            parameters: ['excluded_hosts': hosts.join(',')]
```

Comment:
based on the example in the slack channel, i am not sure if this is needed

Reply:
fixed